state/model.go

```diff
@@ -1292,6 +1292,10 @@ func (m *Model) getEntityRefs() (*modelEntityRefsDoc, error) {
 	return &doc, nil
 }
 
+func (model *Model) State() *State {
```

Comment:
Add a comment and also a TODO that is temporary and will go once refactoring is complete